src/App.jsx

```diff
@@ -14,33 +16,147 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const SORT_ALPHABETICALLY = 'alphabetically';
+  const SORT_BY_LENGTH = 'sort by length';
+  const REVERSE = 'reverse';
+  const RESET = 'reset';
+  const [sort, setSort] = useState('');
+  const [reversePressed, setreversePressed] = useState(false);
```

Comment:
The state variable `setreversePressed` should be named `setReversePressed` to follow the camelCase convention.